core/src/main/java/org/apache/spark/shuffle/sort/UnsafeShuffleWriter.java

```diff
@@ -82,6 +82,7 @@
   private final int initialSortBufferSize;
   private final int inputBufferSizeInBytes;
   private final int outputBufferSizeInBytes;
+  private final int stageAttemptId;
```

Comment:
`shuffleGenerationId`?

Reply:
Sorry for missing, also do the reorder in 4c4af33.